core/optionsx.h

```diff
@@ -294,7 +294,11 @@
 #endif
 
 #ifdef DEADLOCK_AVOIDANCE
+# ifdef STANDALONE_UNIT_TEST
```

Comment:
Sorry, my suggestion of changing the default here will not work -- this define is only for unit tests, while this needs to be done for standalone mode in general which has no separate define.